src/Tizen.NUI.Components/PreloadStyle/OverlayAnimationButtonStyle.cs

```diff
@@ -33,7 +33,9 @@ public class OverlayAnimationButtonStyle : ButtonStyle
         public OverlayAnimationButtonStyle() : base()
         {
             CornerRadius = 10;
-            BackgroundColor = new Color(0.3f, 0.3f, 0.3f, 0.5f);
+            Color color = new Color(0.3f, 0.3f, 0.3f, 0.5f);
+            BackgroundColor = color;
+            color.Dispose();
```

Comment:
Ditto.
Implicit type conversion from `Color` to `Selector<Color>`.
Please fix like this.
```
BackgroundColor = new Selector<Color> { All = new Color(0.3f, 0.3f, 0.3f, 0.5f) };
```

Reply:
Done

Reply:
Done